lib/takeaway.rb

```diff
@@ -0,0 +1,86 @@
+require_relative 'menu'
+require_relative 'dish'
+require_relative 'order'
+require 'twilio-ruby'
+
+class Takeaway
```

Comment:
Could you list out the responsibilities of `Takeaway` for me?

For instance: one is "present the menu to the user"

Reply:
- Present the menu to the user
- Take an order from the user
- Present the checkout to the user
- Accept a payment from the user
- Send an sms to the user
- Accept a cancelation from the user

Reply:
Looks like an accurate assessment to me — now read this: https://en.wikipedia.org/wiki/Single_responsibility_principle